pkg/minikube/out/out.go

```diff
@@ -161,9 +161,10 @@ func wantsColor(fd uintptr) bool {
 	}
 
 	term := os.Getenv("TERM")
+	colorTerm := os.Getenv("COLORTERM")
 	// Example: term-256color
-	if !strings.Contains(term, "color") {
-		glog.Infof("TERM=%s, which probably does not support color", term)
+	if !strings.Contains(term, "color") && !strings.Contains(colorTerm, "truecolor") && !strings.Contains(colorTerm, "24bit") && !strings.Contains(colorTerm, "yes") {
```

Comment:
Thank you for the PR. Is there any documentation we can link here on why the values we are testing are only those 4? 
Those names seem a bit magic to me.

Reply:
Some context behind COLORTERM, because it was a new discovery for me as well:

https://gist.github.com/XVilka/8346728

Reply:
Expected values:

https://www.reddit.com/r/archlinux/comments/5hun36/expected_values_for_term_colorterm/